mysqli/mysqli.php

```diff
@@ -1521,11 +1528,11 @@ function mysqli_commit ($link, $flags = 0, $name = null) {}
  * @param string $user The MySQL user name.
  * @param string $password If not provided or NULL, the MySQL server will attempt to authenticate the user against those user records which have no password only.
  * @param string $database If provided will specify the default database to be used when performing queries.
- * @param string $port Specifies the port number to attempt to connect to the MySQL server.
+ * @param int $port Specifies the port number to attempt to connect to the MySQL server.
  * @param string $socket Specifies the socket or named pipe that should be used.
  * @return mysqli|false object which represents the connection to a MySQL Server or false if an error occurred.
  */
-function mysqli_connect ($host = '', $user = '', $password = '', $database = '', $port = '', $socket = '') {}
+function mysqli_connect ($host = '', $user = '', $password = '', $database = '', $port = 0, $socket = '') {}
```

Comment:
This function is an alias of https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.stub.php#L61 so I think default values should be the same